### ISSUE TITLE
Not compatible with VS Code 1.47.2 message for remote SSH installation

### ISSUE THREAD

**Issue by RobCannon** [microsoft/vscode-pull-request-github#2006]
- Extension version: 0.18.1
- VSCode Version: 1.47.2
- OS: Windows 10

Steps to Reproduce:

1. Start SSH session to remote linux server
2. Select 'Install on SSH' button on the extension page for GitHub Pull Requests and Issues

Error message:

[2020-07-16 17:40:08.226] [renderer3] [error] Unable to install extension 'github.vscode-pull-request-github' as it is not compatible with VS Code '1.47.2'.: Error: Unable to install extension 'github.vscode-pull-request-github' as it is not compatible with VS Code '1.47.2'.
    at /home/xxx/.vscode-server/bin/17299e413d5590b14ab0340ea477cdd86ff13daf/out/vs/server/remoteExtensionHostAgent.js:464:109
    at processTicksAndRejections (internal/process/task_queues.js:89:5)

**Comment by MohamedAlFahim** [comment]
Similar thing happened for me. I downloaded the 0.18.1 VSIX file, and tried to "Install from VSIX", but got an error message saying:
> Unable to install extension 'github.vscode-pull-request-github' as it is not compatible with VS Code '1.47.1'.

**Comment by RMacfarlane** [comment]
Version 0.18.0 should work with 1.47.2, but in 0.18.1 the required engine was bumped to the latest insiders version, 1.48.0, to absorb an API change there. So @MohamedAlFahim the behavior you're seeing is expected, please try downloading version 0.18.0 instead.

@RobCannon I'm having trouble reproducing this, if I don't have the extension installed, I connect to a remote, and I select 'Install on SSH', it seems to correctly install 0.18.0 there. Is there anything I'm missing?

**Comment by RobCannon** [comment]
@RMacfarlane, I did not have the extension installed locally.  It defaulted to installing 0.18.1 when I tried to install the extension on in my remote SSH session.

I was able to work around this by installing the extension locally, which installed 0.18.1.  Then, I was able to "Install Another Version..." and could selection version 0.18.0.  THEN, I was able to connect to my SSH instance and it would allow me to install 0.18.0 to the remote session.
